src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -17,25 +21,20 @@ import {
 const selectTradeQuoteSlice = (state: ReduxState) => state.tradeQuoteSlice
 
 // TODO(apotheosis): Cache based on quote ID
-export const selectSelectedQuote = createDeepEqualOutputSelector(
-  selectTradeQuoteSlice,
-  swappers => swappers.quote,
-)
+export const selectSelectedQuote: Selector<ReduxState, TradeQuote | undefined> =
```

Comment:
Interesting type assertion, do we need this if `selectTradeQuoteSlice` is typed? And if we want to strongly type the return type, can't we type it on the line below instead i.e in the callback?

Reply:
I'm not sure if this question is directed at this particular usage, or this PR as a whole, but the key intent of the PR is that these are _not_ type assertions (weakening type checking), they are type _annotations_ to ensure that what we are expecting to be returned by each selector is enforced at compile time.